settings.gradle

```diff
@@ -23,5 +23,5 @@ dependencyResolutionManagement {
         mavenCentral()
     }
 }
-rootProject.name = "Price Converter"
+rootProject.name = "Minimalistic Apps"
```

Comment:
Why this change? Project is called "Price Converter"

Reply:
I changed it because you send the link to the color scheme and you mentioned the name as Minimalistic app that's why I changed it.